setup.cfg

```diff
@@ -34,6 +34,7 @@ install_requires =
     torch>=1.0.0
     uproot3>=3.14.1
     vector>=0.8.4
+    sympy>=1.10.1
```

Comment:
Is this version of `sympy` known to be the oldest version that will still work? Was part of the `sympy` API that is used in this PR added in `v1.10.1`?

Reply:
It's the version that I used in my environment. I assume there are older versions that can work, but haven't tested yet. I'll try to test it sometime later.

Reply:
Yeah it would be good to not be overly restrictive when it comes to core dependencies.